src/database/migrations/create-9-reaction.js

```diff
@@ -0,0 +1,59 @@
+export default {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('Reactions', {
+      id: {
+        allowNull: false,
+        primaryKey: true,
+        type: Sequelize.UUID,
+        defaultValue: Sequelize.literal('uuid_generate_v4()')
+      },
+      accomodationId: {
+        type: Sequelize.DataTypes.UUID,
+        allowNull: false,
+        references: {
+          model: 'Accommodations',
+          key: 'id',
+        },
+        onUpdate: 'CASCADE',
+        onDelete: 'SET NULL'
+      },
+      userId: {
+        type: Sequelize.DataTypes.UUID,
+        allowNull: false,
+        references: {
+          model: 'Users',
+          key: 'id',
+        },
+        onUpdate: 'CASCADE',
+        onDelete: 'SET NULL'
+      },
+      likes: {
+        type: Sequelize.INTEGER,
+        allowNull: false,
+        defaultValue: 0
+      },
+      dislikes: {
+        type: Sequelize.INTEGER,
+        allowNull: false,
+        defaultValue: 0
+      },
+      reactionType: {
```

Comment:
Why having a `reactionType` column,  when you already have columns for likes and dislikes

Reply:
The `reactionType` is the user reaction itself. either `like` or `unlike` while `likes` and `dislikes` are *counts* of the likes and unlikes

Reply:
Why do we need a `like` and `unlike` column? We can have one column called `reaction` that stores a true or false value instead of three separate columns. To get the number of likes you count the number of true reactions and to get dislikes you count the false reactions for a particular accommodation. I still don't think storing user values and then still storing the words like or dislike is a good structure

Reply:
I get your point. I was thinking of a way to prevent a user from showing the same reaction more than once. That was why I came up with that structure. But I think it still works with what you said. I'll see if it works with how I plan to implement it. Thank you